atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueService.java

```diff
@@ -1290,32 +1290,29 @@ public void dropTables(final Set<TableReference> tablesToDrop) {
 
     private void dropTablesInternal(final Set<TableReference> tablesToDrop) throws Exception {
         try {
-            clientPool.runWithRetry(new FunctionCheckedException<Client, Void, Exception>() {
```

Comment:
Generally this should be avoided in a PR with changes.  Makes it significantly harder to parse.